articles/metadata/index.md

```diff
@@ -27,29 +28,29 @@ Suppose the following data is stored for a user with the email address "jane.doe
 }
 ```
 
-**Note**: Any valid JSON snippet may be used as metadata.
+**NOTE**: Any valid JSON snippet can be used as metadata.
 
-To read metadata, simply access the correct property. For example, suppose you wanted to work with the values of the following properties in your Rules or via a call to the Management API:
+To read metadata, simply access the correct property. For example, if you want to work with the values of the following properties in your Rules or via a call to the Management API:
 
 ```js
 console.log(user.email); // "jane.doe@example.com"
 console.log(user.user_metadata.hobby); // "surfing"
 console.log(user.app_metadata.hobby); // "full"
 ```
 
-> **Note**: With APIv1, all metadata was stored in the `metadata` field. Data stored in this field will now be available under `app_metadata`.
+**NOTE**: With APIv1, all metadata was stored in the `metadata` field. Data stored in this field is now available under `app_metadata`.
 
 ### Naming Metadata Fields
 
-Metadata field names must not contain a dot. For example, use of the following returns a Bad Request (400) error:
+Metadata field names must not contain a period. For example, use of the following returns a Bad Request (400) error:
```

Comment:
This is nitpicking it a little bit, but `dot` is not wrong in this case. A period is, as far as I'm aware, a synonym for the full stop. Thus it stands for a *dot* at the end of a sentence. In a metadata field, I would consider *dot* to be correct, as we are not dealing with a sentence. <- period :-)